### PR TITLE
LPS-146004 [CP2.0] - Add CP2ActivationKeysForCommerce tests

### PR DESCRIPTION
**Pull request by MathausC**
https://issues.liferay.com/browse/LPS-146004

### CODE REVIEW

**Comment by MathausC** [comment]
ci:test:sf

**Comment by MathausC** [comment]
ci:test:osb-site-initializer-customer-portal

**Comment by MathausC** [comment]
@barrosdennys needs review.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro [hunk 1]
```diff
@@ -0,0 +1,21 @@
+definition {
+
+	macro assertDropDownSelection {
+		var key_option = "${option}";
+
+		AssertElementPresent(locator1 = "CP2ActivationKeys#DROPDOWN");
+
+		AssertVisible(locator1 = "CP2ActivationKeys#DROPDOWN");
```
[the remark this answers is not in the document]
Comment (MathausC) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro:8]: @barrosdennys I put this to guarantee that the element is present to check if it's visible, but I think the function AssertVisible already do that. I'll remove the AssertVisible statement.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	}
+
+	macro assertInstructionsTable {
+		for (var text : list "${textList}") {
+			var key_text = "${text}";
+
+			AssertElementPresent(locator1 = "CP2ActivationKeys#RESPONSIVE_TABLE");
+
+			AssertVisible(locator1 = "CP2ActivationKeys#RESPONSIVE_TABLE");
```
[the remark this answers is not in the document]
Comment (MathausC) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro:17]: Same as the comment above.

**Comment by MathausC** [comment]
ci:test:sf

**Comment by MathausC** [comment]
@barrosdennys changes made.

**Comment by MathausC** [comment]
ci:test:osb-site-initializer-customer-portal

**Comment by MathausC** [comment]
ci:test:sf

**Comment by MathausC** [comment]
ci:test:sf

**Comment by MathausC** [comment]
ci:stop

**Comment by MathausC** [comment]
ci:test:sf

**Comment by MathausC** [comment]
ci:test:osb-site-initializer-customer-portal

**Comment by MathausC** [comment]
ci:test:osb-site-initializer-customer-portal

**Comment by MathausC** [comment]
ci:run:stop

**Comment by MathausC** [comment]
ci:test:stop

**Comment by MathausC** [comment]
ci:stop

**Comment by MathausC** [comment]
ci:test:osb-site-initializer-customer-portal:nocompile

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2FileUtil.macro [hunk 3]
```diff
@@ -0,0 +1,18 @@
+definition {
+
+	macro assertDownloadedFileIsPresent {
+		var tempDir = PropsUtil.get("output.dir.name");
+
+		var downloadedFilePath = "${tempDir}/${downloadedFileName}";
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2FileUtil.macro:6]: @MathausC I was wondering here, when we download something via browser, isn't the file downloaded going to the default path defined in the browser settings? I was just wondering if this directory really works. If you confirmed it, please disregard this comment.

**Comment by MathausC** [comment]
ci:test:sf

**Comment by MathausC** [comment]
ci:test:sf

**Comment by MathausC** [comment]
ci:test:relevant

**Comment by MathausC** [comment]
ci:test:relevant

**Comment by MathausC** [comment]
ci:test:relevant

**Comment by MathausC** [comment]
ci:forward

**Comment by MathausC** [comment]
ci:stop

**Comment by MathausC** [comment]
ci:test:relevant

**Comment by MathausC** [comment]
ci:forward

**Pull request closed without merging**